### DISCUSSION TITLE
In the feedback assessment, are all questions mandatory to anwser?

### DISCUSSION THREAD

**Discussion by maeste** [open-organization/open-leadership-assessment#12] [General]
While I was converting the feedback assessment into a google form I made all questions as mandatory, but then I got the doubt I was right? Are all questions intended to be mandatory? IOW we suppose the person who we are asking the feedback is able to assess all 50 questions, or is a "I don't know" answer somehow permitted?

Best Regards
Stefano

**Comment by HeidiHVL** [comment]
Hi there @maeste/Stefano, 

Thanks for the question. 

We recommend that the questions are mandatory and that there is no "I don't know" option. This is because the questionnaire is intended to be scored at the end. Being able to score on a complete set of 10 questions means that we can compare one principle to another (I scored 10% on transparency and 70% on community, for instance) as well as have greater consistency scoring between the self-assessment vs team assessment (that is, the leader self-scored 60% on collaboration and the team scored the leader as 30% on collaboration). If people can skip questions or say "i don't know" then scoring can't be compared. 

Of course, this is just what we recommend. :) It's up to you, as the user, to determine whether you want to allow those options and whether you want to compare scores between assessments or across principles. 

I hope this helps! 
Heidi

**Comment by maeste** [comment]
I see your point. Just to give you feedback on how people feel getting to this survey I received at the moment 2 answers and I got these 2 comments:

"I don't mind being asked for feedback and I'm happy providing it, though this form definitely have too many questions; moreover, all questions are mandatory, while most of those who are asked to provide feedback might not have experimented all the situations described in th e questions, so answering becomes difficult."

"Lower scores are in some cases where I don't really know the answer :-) Keep up the good work!"

The problem with all mandatory is in some case answer are almost random or at least not well supported by real experience or strong opinion

I also see the point of the first comment: 50 questions with a 1/5 assessment are really a lot. Maybe we can try to elaborate a short er version as an alternative for quick feedback? Or maybe 2 different scores, one only considering mandatory questions and one for full answers? The risk with these 50 mandatory questions is to lose valuable feedback because is too long to provide
